test/routes_test.go

```diff
@@ -0,0 +1,27 @@
+package test
```

Comment:
No unit tests in /test  (sorry should have called it "e2e_test" or something - I think this test already exists in client/call_fn_test.go (my bad)